baseplate/lib/experiments/variant_sets/rollout_variant_set.py

```diff
@@ -12,28 +17,29 @@ class RolloutVariantSet(VariantSet):
     going from 55% to 45% will result in only 10% of users losing the
     treatment.
 
-    :param list variants: array of dicts, each containing the keys 'name'
+    :param variants: array of dicts, each containing the keys 'name'
         and 'size'. Name is the variant name, and size is the fraction of
         users to bucket into the corresponding variant. Sizes are expressed
         as a floating point value between 0 and 1.
-    :param int num_buckets: the number of potential buckets that can be
+    :param num_buckets: the number of potential buckets that can be
         passed in for a variant call. Defaults to 1000, which means maximum
         granularity of 0.1% for bucketing
 
     """
 
-    def __init__(self, variants, num_buckets=1000):
+    # pylint: disable=super-init-not-called
```

Comment:
should we just go ahead and call `super` or is there a reason we didn't?

Reply:
super raises `NotImplementedError`!

Reply:
(i'm not fond of this, but it was the simplest way I could find to type the dictionary mapping their names to types -- open to suggestions)